server/portal/apps/workspace/api/urls.py

```diff
@@ -11,6 +11,7 @@
         r'^jobs/(?P<job_uuid>[0-9a-fA-F]{8}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{4}\-[0-9a-fA-F]{12}\-[0-9a-fA-F]{3})/history/?$',
         views.JobHistoryView.as_view()
     ),
+    url(r'^jobs/(?P<operation>\w+)/?$', views.JobsView.as_view()),
     url(r'^jobs/?', views.JobsView.as_view()),
```

Comment:
Is line 15 needed now that line 14 is added?

Reply:
Yeah this is where I was running into a bit of an issue and could use some feedback. With only line 14 and not line 15 the POST (and probably DELETE) endpoints were not being triggered properly because currently those urls don't have the `operation` parameter on them. One solution would be to add the other url as I added above or add the `operation` parameter to POST and DELETE as well. It shouldn't be a huge change if we were to go for the latter.